src/cpp/session/modules/SessionObjectExplorer.R

```diff
@@ -444,6 +453,25 @@
       .rs.explorer.inspectDefault(object, context)
 })
 
+.rs.addFunction("explorer.inspectReticulateObject", function(object,
+                                                             context = .rs.explorer.createContext())
+{
+   children <- NULL
```

Comment:
Is there a way of accessing these objects without automatically marshaling them back to R?